sdk/src/main/scala/com/horizen/account/receipt/LogsBloom.scala

```diff
@@ -20,25 +25,24 @@ class LogsBloom() {
     log.topics.foreach(topic => addBytesToBloomFilter(topic.toBytes))
   }
 
-  def addBytesToBloomFilter(
-      data: Array[Byte]
-  ): Unit = {
+  def addBytesToBloomFilter(data: Array[Byte]): Unit = {
     val (bloomFilterIndexes, bloomFilterValues) = getBloomFilterValues(data)
 
     bloomFilterIndexes.zipWithIndex
       .foreach({ case (bloomFilterIndex, i) =>
-        bloomFilter(bloomFilterIndex) =
-          (bloomFilter(bloomFilterIndex) | bloomFilterValues(i)).toByte
+        bloomFilter(bloomFilterIndex) = (bloomFilter(bloomFilterIndex) | bloomFilterValues(i)).toByte
       })
   }
 
-  def addBloomFilter(bloomFilter: LogsBloom): Unit = {
+  def addBloomFilter(bloomFilter: LogsBloom): LogsBloom = {
```

Comment:
Let's keep the result type as Unit.
Having `LogsBloom` lead to the idea that addBloomFilter remain the existing instance immutable - which is not our case.
In the forger reuse `fromEthereumReceipt` that should take `EthereumConsensusDataReceipt` seq

Reply:
Ok, I added that because it allowed me to use reduce on `EthereumReceipt` seq that was created by `map` from `EthereumConsensusDataReceipt` seq. Should I add another method called `fromEthereumConsensusDataReceipt` to the `LogsBloom` or I should extract `LogsBloom` seq from `EthereumConsensusDataReceipt` to be able to use `fromEthereumReceipt`. Also, approach would be to initialize a new empty `LogsBloom` and add receipts (by calling `addBloomFilter`), one by one to it.

Reply:
"or I should extract LogsBloom seq from EthereumConsensusDataReceipt to be able to use fromEthereumReceipt. After that, approach would be to initialize a new empty LogsBloom and add receipts (by calling addBloomFilter), one by one to it." I vote for this one